src/main/java/de/adesso/kicker/notification/tournamentjoinrequest/TournamentJoinRequestService.java

```diff
@@ -0,0 +1,96 @@
+package de.adesso.kicker.notification.tournamentjoinrequest;
+
+import de.adesso.kicker.notification.NotificationRepository;
+import de.adesso.kicker.team.Team;
+import de.adesso.kicker.tournament.Tournament;
+import de.adesso.kicker.tournament.singleelimination.SingleElimination;
+import de.adesso.kicker.tournament.singleelimination.SingleEliminationService;
+import de.adesso.kicker.user.User;
+import de.adesso.kicker.user.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TournamentJoinRequestService {
+
+    private NotificationRepository notificationRepository;
+    private UserService userService;
+    private SingleEliminationService singleEliminationService;
+
+    @Autowired
+    public TournamentJoinRequestService(NotificationRepository notificationRepository, UserService userService,
+            SingleEliminationService singleEliminationService) {
+
+        this.notificationRepository = notificationRepository;
+        this.userService = userService;
+        this.singleEliminationService = singleEliminationService;
+    }
+
+    public void acceptTournamentJoinRequest(long notificationId) {
+
+        if (!notificationRepository.existsById(notificationId)) {
+            System.err.println(
+                    "ERROR at 'TournamentJoinRequestService' --> 'acceptTournamentJoinRequest()' : Cannot find TournamentJoinRequest with id "
+                            + notificationId);
+            return;
```

Comment:
Don't do this. Could have put the other code into an else block. Also the null check is unnecessary.

Reply:
Yes in such a case the test will fail. That was the reason that we created all those null checks

Reply:
But there shouldn't be a real world case where there would be a null.